weblate/api/tests.py

```diff
@@ -1148,6 +1150,74 @@ def test_create(self) -> None:
         )
         self.assertEqual(Project.objects.count(), 2)
 
+    def test_create_with_billing(self) -> None:
+        installed_apps = settings.INSTALLED_APPS
+        if "weblate.billing" not in installed_apps:
+            installed_apps = ["weblate.billing", *settings.INSTALLED_APPS]
+        with override_settings(INSTALLED_APPS=installed_apps):
```

Comment:
Using `modify_settings` instead would make this code simpler.

Reply:
Done.